client/src/components/modals_and_popovers/FixedPopover.stories.js

```diff
@@ -0,0 +1,80 @@
+import { useArgs } from "@storybook/client-api";
+import React, { Fragment } from "react";
+
+import { FixedPopover } from "./FixedPopover";
+
+export default {
+  title: "modals and popovers/FixedPopover",
+  component: FixedPopover,
+
+  // Need decorators to use useArgs()
+  decorators: [(Story) => <div>{Story()}</div>],
+};
+
+const Template = (args) => {
+  const [_, updateArgs] = useArgs();
+
+  function on_close_callback() {
+    console.log(args.show ? "Closing " : "Opening " + " modal.");
+    updateArgs({ ...args, show: !args.show });
+  }
+  return (
+    <Fragment>
+      <div id="ib-site-header-area" />
+      <div style={{ height: "500vh" }}> Scroll down... </div>
+      {args.show ? null : (
+        <button onClick={on_close_callback}>Click to open modal</button>
+      )}
```

Comment:
Hard to find, maybe make it a child of the div with `height: "500vh"`

Reply:
Done!